src/main/java/seedu/exchangecoursemapper/command/AddCoursesCommand.java

```diff
@@ -48,7 +62,7 @@ public void execute(String userInput, Storage storage) {
                 logger.log(Level.INFO, Logs.ADD_APPROVED_MAPPING);
                 Course courseToStore = new Course(puCourse, nusCourse, pu);
                 storage.addCourse(courseToStore);
-                printAddMessage(courseToStore);
+                ui.printAddMessage(courseToStore);
```

Comment:
I like that you made the code more OOP by moving the user interaction methods to the UI class! 👍